app/models/item.rb

```diff
@@ -18,4 +18,12 @@ def self.enabled_only
   def self.disabled_only
     where(enabled: false)
   end
+
+  def self.ready_to_ship
+    joins(invoices: :invoice_items)
+    .select("items.*, invoices.id AS invoice_id, invoices.created_at AS invoice_creation")
```

Comment:
It's possible that similarly here, since you aliased, you might be able to `group(:id, :invoice_id)` though I'm not sure (just curious)

Reply:
Just tried it, but it didn't work..Here's the error: PG::AmbiguousColumn: ERROR:  column reference "invoice_id" is ambiguous

Reply:
Ahhh because it is the same as the items.invoice_id column. Makes sense.